packages/bot/src/scratch/utils/scratch-helper.js

```diff
@@ -0,0 +1,22 @@
+import config                   from '../../constants';
```

Comment:
Can we move this logic to `scratch/utils.js`? Or if you want to organise the utilities, a name such as `workspace.js` would make more sense imo, this could be placed in `scratch/utils/` folder.